### DISCUSSION TITLE
Leggere codici IR

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2506] [GENERALE - Discussioni generiche]
Ciao a tutti,
ho installato un Broadlink per emulare i telecomandi dei clima, ovviamente non c'è il codice per il mio modello ma non trovo il modo di leggere il codice/i codici per poi creare il file di configurazione.
l'app e-control non mi va e nemmeno broadlink manager, quest'ultimo vede il dispositivo ma lo calssifica come non riconoscuto.

**Comment by Matty1184** [comment]
dimenticavo il broadlink è un RM4 mini

**Comment by KC-inDomus** [comment]
usa home assistant
https://indomus.it/formazione/come-usare-gli-attuatori-broadlink-su-home-assistant/

**Reply by Matty1184** [reply]
> usa home assistant
> 
> https://indomus.it/formazione/come-usare-gli-attuatori-broadlink-su-home-assistant/

Il problema che usando il servizio di home assistant salva sempre il comando... considerato che devo costruire il file per il telecomando del condizionatore ho tanti codici da copiare
